src/utils/backendAuthMixin.js

```diff
@@ -0,0 +1,47 @@
+import { mapState } from 'vuex';
+import Backend from './backend';
+import { createDeepLinkUrl } from './util';
+import { client } from './aeternity';
+
+export default (ignoreCallInQuery) => ({
+  computed: mapState(['useSdkWallet', 'account']),
+  ...!ignoreCallInQuery && {
+    async beforeRouteEnter(to, from, next) {
+      const {
+        method, address, challenge, signature,
+      } = to.query;
+      if (!method || !address || !challenge || !signature) {
+        next();
+        return;
+      }
+      let succeed = false;
+      try {
+        await Backend[method](address, { challenge, signature });
```

Comment:
I'm proposing to postpone rendering until the request is complete to don't refresh data after initial load